frontend/src/styles/HetComponents/HetDivider.tsx

```diff
@@ -0,0 +1,12 @@
+// @BEN: could export MUI Divider here instead if you'd prefer
+const HetDivider = ({ className }: { className?: string }) => {
+  return (
+    <hr
+      className={`m-0 flex-shrink-0 border-0 border-b border-solid border-dividerGrey ${
+        className ?? ''
+      }`}
+    />
+  )
+}
+
+export default HetDivider
```

Comment:
hm... this way is good! really the thing im after is consistency, so that future devs will know "i need a horizontal rule divider... this is where i'll go because the rest of the codebase uses this" 

tiny nit pick that is totally a personal opinion, but I am desperately trying to make the entire codebase consistent with this as well... I personally really dislike declaring functions `const` unless it's truly needed (like needing `this` or similar). For this app, for react components, I really prefer if our components follow the same syntax and pattern as it reduces cognitive burden when jumping around to components you don't touch often or even have never seen. if they all follow the same pattern and syntax it just makes things a little easier to navigate, and much easier to refactor / find and replace, etc.

```typescript
interface MyComponentProps {
// props
}

export default function MyComponent(props: MyComponentProps){
return (
   <>My Component</>
)
}
```

Reply:
ah of course, ok - I will continue with this pattern from now on - my prev PR's need these changes as well